src/components/prescriptions/PrescriptionComponent.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+    <li
+        v-bind:key="props.id"
+        class="documentBox">
+        <font-awesome-icon :icon="classOfIcon" />
+        <h5 class="documentBoxtitle">{{ name }}</h5>
+        <h5 class="documentBoxtitle">{{ date }}</h5>
+        <button-component
+            small
+            @click="openModalDetails = true">
+            Szczegóły
+        </button-component>
+        <teleport to="body">
+            <transition name="modal">
+                <div
+                    v-if="openModalDetails"
+                    class="modal">
+                    <modal-component
+                        ref="modalDetils"
+                        @close="openModalDetails = false">
+                        <prescription-details
+                            v-bind:key="detailsList[index].id"
```

Comment:
you add key but you dont render elements in v-for. Maybe is it better to render in loop list of elements and passing props. This is dangerous because what about empty array? There could be a problem with render whole view.